qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -86,6 +86,8 @@
             no Windows
             gagent_check_type = fsfreeze
             gagent_fs_test_cmd = "rm -f /tmp/foo; echo foo > /tmp/foo"
+        - check_reset_reboot_shutdown_fsfreeze:
```

Comment:
This case supports isa_serial, As virtio_serial and isa_serial have different return value for agent command "guest-shutdown" when guest fsfreeze. e.g virtio_serial return:return {}, isa_serial result: "The command guest-shutdown has been disabled for this instance".  This case will work for isa_serial once avocado fix this problem.  I will raise this problem to maintainer.